src/Core/Managed/Shared/Extensibility/Implementation/Property.cs

```diff
@@ -141,8 +141,15 @@ public static void SanitizeMeasurements(this IDictionary<string, double> diction
 
                     string sanitizedKey = SanitizeKey(entry.Key, dictionary);
 
+                    // Disallow Nan and Infinity since Breeze does not accept it
+                    double sanitizeValue = entry.Value;
```

Comment:
do we have any other double values anywhere? I believe Metric may have it and Measurements colleciton. Worth checking maybe we have anything else that potentially can become NaN